Datastore/src/Operation.php

```diff
@@ -614,12 +750,40 @@ public function commit(array $mutations, array $options = [])
             'databaseId' => $this->databaseId,
         ];
 
-        $res = $this->connection->commit($options + [
-            'mode' => ($options['transaction']) ? 'TRANSACTIONAL' : 'NON_TRANSACTIONAL',
+        foreach ($mutations as &$mutation) {
```

Comment:
Extract this out in a prvt method?

Reply:
This logic is specific to parsing mutations exclusively in commit rpc. Does extracting makes sense?

Reply:
In this case we want to extract it more for readability instead of DRY,.

Reply:
Done in https://github.com/yash30201/google-cloud-php/pull/7